### PR TITLE
The dashboard page is finished (mostly) and styled.

### PR DESCRIPTION
**Pull request by AlexJCarmichael**
I want to get this code merged and pushed to Heroku so I can do some real testing with the Ipad. Missing elements so far:

Footer: Nagmeh is working on this.
Time display: Just yuck.
The design: I am not sold on and would like some input. A big change I think I need is to change the buttons but I just don't have the bandwith at the moment.
The container: Need some thoughts from the Bootstrap people. I don't like how narrow the container is, especially as it expands, have you all messed with the container settings yet? It is annoying me because once I switch from Ipad size the columns have to shrink roughly 12 em in order to have a nice float.

Need to turn all of the events into link_to's to the show page. Was going to wait until we made the show page to do it.

Lastly, LOL the time estimate amirite?

### CODE REVIEW

**Comment by Naghmehd** [comment]
👍

File: app/views/dashboard/home.html.erb [hunk 1]
```diff
@@ -1 +1,33 @@
...
+  <ul class='pending-events-dispaly-list'>
+    <% @finished.each do |finish| %>
+      <li class='pending-events-dispaly-list-item'><%= finish.name.titleize %> | <%= finish.agenda %> | <%= finish.formatted_start_time %> until <%= finish.end_time %></li>
+    <% end %>
+  <ul class='pending-events-dispaly-list'>
+  </ul>
+</div>
+
+<div class='pending-events-dispaly'>
+  <h5><%= link_to "Create an event?", campus_path(current_user.campus), class: "dashboard-event-create" %></h5>
```
Comment (VanAnderson) [inline app/views/dashboard/home.html.erb:32]: Will we have a create event form on the campus show page? If not, campus_path(current_user.campus) should maybe be changed.

File: app/models/event.rb [hunk 2]
```diff
@@ -15,9 +15,13 @@ class Event < ActiveRecord::Base
   belongs_to :user
   validates_with UniqueTime
 
-
   def formatted_start_time
-    self.start_time
+    "#{start_time.strftime('%x')} at #{start_time.strftime('%r')}"
```
Comment (VanAnderson) [inline app/models/event.rb:19]: :+1:  Works good for me!

**Comment by VanAnderson** [comment]
:+1:  good job! I merged and tested this locally and couldn't find any bugs.

**Pull request merged**